src/ui/components/search/locationbiascomponent.js

```diff
@@ -32,7 +34,16 @@ export default class LocationBiasComponent extends Component {
      * Recieve updates from storage based on this index
      * @type {StorageKey}
      */
-    this.moduleId = StorageKeys.LOCATION_BIAS;
+    this.core.storage.registerListener({
```

Comment:
Are there any possible negative side effects of not setting `moduleId` anymore? I doubt anyone is dependent on it, but i'm just curious.

Reply:
I think the main difference is that when you remove a component, we automatically deregister the moduleId listener for you. We don't do any of that for listeners you register manually. This only really matters for components you would be creating and removing a ton (mostly child components)